great_expectations/cli/datasource.py

```diff
@@ -1234,112 +1242,128 @@ def _get_batch_kwargs_from_generator_or_from_file_path(
     return (data_asset_name, batch_kwargs)
 
 
+def _get_default_schema(datasource):
+    inspector = sqlalchemy.inspect(datasource.engine)
+    return inspector.default_schema_name
+
+
 def _get_batch_kwargs_for_sqlalchemy_datasource(
     context, datasource_name, additional_batch_kwargs=None
 ):
-    msg_prompt_query = """
-Enter an SQL query
+    data_asset_name = None
+    sql_query = None
+    datasource = context.get_datasource(datasource_name)
+    msg_prompt_how_to_connect_to_data = """
+You have selected a datasource that is a SQL database. How would you like to specify the data?
+1. Name the table and schema I am interested in
+2. Input a custom SQL query
+3. Search for the table to add (will display all tables in database)
```

Comment:
Looking for feedback on this message. Is there anything you'd like to adjust?

Reply:
@Shinnnyshinshin from the end user perspective, maybe we can make the message a little more descriptive e.g.

```
1. Name the table and schema I am interested in (SELECT from a single table)
2. Input a custom SQL query (with additional SQL clauses e.g. WHERE/JOIN/GROUP BY)
3. Search for the table to add (will display all tables in database)
```

Reply:
Thanks @armaandhull :) I can adjust this now

Reply:
@Shinnnyshinshin I suggest a change in the prompt:

1. Enter a table name and schema 
2. Enter a custom SQL query
3. List all tables in the database (this may take a very long time)